File: app/src/main/java/ashcollauthlogin/darkzek/com/IntroAdapter.java
package ashcollauthlogin.darkzek.com;

import android.graphics.Color;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

public class IntroAdapter extends FragmentPagerAdapter {

    public IntroAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        if (position % 2 == 0) {
            return IntroFragment.newInstance(Color.parseColor("#03A9F4"), position); // blue
        } else {
            return IntroFragment.newInstance(Color.parseColor("#4CAF50"), position); // green
        }
    }

    @Override
    public int getCount() {
        return 3;
    }

}

File: app/src/main/java/ashcollauthlogin/darkzek/com/IntroAdapter.kt
package ashcollauthlogin.darkzek.com

import android.graphics.Color
import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter

class IntroAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    override fun getItem(position: Int): Fragment {
        when (position) {
            0 -> return IntroFragment.newInstance(Color.parseColor("#03A9F4"), position) // blue
            1 -> return IntroFragment.newInstance(Color.parseColor("#4CAF50"), position) // green
            2 -> return IntroFragment.newInstance(Color.parseColor("#df6142"), position) // orange
        }

        return IntroFragment.newInstance(Color.parseColor("#df6142"), position) // orange
    }

    override fun getCount(): Int {
        return 3
    }

}